fastlane/Pluginfile

```diff
@@ -6,7 +6,9 @@ group :screenshots, optional: true do
   gem 'rmagick', '~> 3.2.0'
 end
 
-gem 'fastlane-plugin-wpmreleasetoolkit', '~> 1.3.1'
+gem 'fastlane-plugin-wpmreleasetoolkit',
+  git: 'git@github.com:wordpress-mobile/release-toolkit',
+  branch: 'add-skip-deliver-in-ios-hotfix-version-bump'
```

Comment:
Leaving this PR a draft till https://github.com/wordpress-mobile/release-toolkit/pull/287 lands on the main branch and a new version of the release toolkit ships